content/pop-recovery.md

```diff
@@ -19,23 +19,52 @@ section: software-troubleshooting
 
-> **NOTE:** These instructions assume Pop!\_OS is the only OS running on your system. If you are booting more than one operating system you may need to change your boot order first, or manually select the Pop!\_OS Disk from your BIOS/Boot menu.
+ **NOTE:** These instructions assume Pop!\_OS is the only OS running on your system. If you are booting more than one operating system you may need to change your boot order first, or manually select the Pop!\_OS Disk from your BIOS/Boot menu.
 
 Once the menu is shown, choose **Pop!_OS Recovery**.
 
 ![systemd-boot](/images/pop-recovery/systemd-boot.png)
 
+## Clean Install
+
+This option erases the current install along with all user files. It reformats the drive partitions and installs the version of Pop!\_OS contained in the Recovery partition.
+
+Steps to back up user-files from a Live Disk/Recovery can be found [here](https://support.system76.com/articles/disaster-recovery)
+
+**NOTE:**
+The Recovery partition OS version will either be the same as the OS version that shipped with your computer, or whichever version to which the Recovery partition has been [updated](#update-recovery-partition)
+
+## Refresh Install
+
+The Refresh Install option allows you to reinstall the OS without losing user account information and data in the home directory. If the `Refresh Install` option is not present on the install screen, one of two things may be true.
+
+1. Your drive is encrypted. The Refresh install option may appear after decrypting the drive. A notice about decrypting the drive will be present above the install options.
+
+2. The Recovery version is out of date. See the [update instructions](#update-recovery-partition)
+
+**NOTE:** user-installed applications are not preserved and will need to be reinstalled.
+
+![Refresh Install Option](/images/pop-recovery/recovery-install-page-20.04.png)
+
+### Reinstall
+
+Once Recovery has booted, the <u>Pop Installer</u> will start automatically.  If the system needs to be reinstalled, go ahead and continue the installation steps as demonstrated [here](/articles/install-pop/).
+
+If the existing install is encrypted, please see the [encrypted disk](#encrypted-disk) instructions.
+
 ## Repair
 
-If the existing system needs to be repaired, then click the **Install Pop!_OS** in the top left, and choose **quit**.
+If the existing system needs to be repaired, then click the **Install Pop!_OS** in the top left, and choose **quit**, or choose “try demo mode” after the language and keyboard selection screens.
+
+**NOTE:** be sure not to choose any install or repair options, as this could result in data loss.
 
 To access to the existing OS drive and run the package manager [repair commands](/articles/package-manager-pop/), the following commands will need to be run:
```

Comment:
This sounds incorrect based on the grammar.

Reply:
Re-working this to:

"To access an existing OS drive, run the following commands...

Once the drive is accessed, commands for maintenance can be run on the installed system. For example, [package manager repair commands](/articles/package-manager-pop/)."

Reply:
This has now been edited for clarity. Commit incoming.